docs/conf.py

```diff
@@ -56,6 +58,25 @@
 html_theme = 'pydata_sphinx_theme'
 # html_static_path = ['_static']
 
+# -- manage thumbnails --------------------------------------------------------
+# define thumbnails
+# (must be in 'resources' and will always be copied to'_images')
+nbsphinx_thumbnails = {
+    'gallery/interactive/pyfar_audio_objects': '_images/thumbnail_pyfar_audio_objects.png',
+}
+
+# create _image folder if it does not exist
+path = ''
+for folder in ['_build', 'html', '_images']:
+    path = os.path.join(path, folder)
+    if not os.path.exists(path):
+        os.mkdir(path)
+
+# copy thumbnails (they are assumed to be located in resources folder)
+for thumbnail in nbsphinx_thumbnails.values():
+    thumbnail = thumbnail.split('/')[-1]
+    shutil.copyfile(os.path.join('resources', thumbnail),
+                    os.path.join(path, thumbnail))
```

Comment:
```suggestion
```
I think this can be deleted not. I'd prefer using Sphinx mechanisms when possible here